WolvenKit.App/Helpers/DiscordHelper.cs

```diff
@@ -6,48 +6,46 @@ namespace WolvenKit.App.Helpers;
 public static class DiscordHelper
 {
     public static DiscordRPC.DiscordRpcClient? Client;
-    private const string s_discordAppID = "897813293480169532";
+    public static string DiscordAppID = "897813293480169532";
     public static bool DiscordRPCEnabled = true;
-    private static bool s_discordRPCInitialized = false;
+    public static bool DiscordRPCInitizialized = false;
+    public static DateTime Begin;
 
     public static void InitializeDiscordRPC()
     {
-        if (DiscordRPCEnabled != true || s_discordRPCInitialized != false)
+        if (DiscordRPCEnabled == true && DiscordRPCInitizialized == false)
```

Comment:
I'd appreciate it a lot if you'd keep the early return, since it makes reading nested code easier :)